examples/legacy_basic_reconnect.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import time
+from niveristand.errors import RunError
+from niveristand.legacy import NIVeriStand
+
+# This is an example script to demonstrate the usage of NIVeriStand Python API.
+# This script requires Python 32b(tested on Python 3.9.6) and NIVeriStand Python API.
+# It can be downloaded here: https://niveristand-python.readthedocs.io/en/latest/index.html
+
+# This script deploys example project on the selected GATEWAY and it prints two channel values every 100ms
+# This script requires VeriStand serwer to be running. It is ran automatically after launching VeriStand.
```

Comment:
We can easily launch the server in this script, making this example more easy to run. You just need to run 
```
NIVeriStand.LaunchNIVeriStand()
NIVeriStand.WaitForNIVeriStandReady()
```
See mix_legacy_and_rtseq_run for an example.

Reply:
True that, but this example should have the SDF already deployed.